sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_stats_clients_frst_seen_v2/query.sql

```diff
@@ -0,0 +1,167 @@
+-- Get all clients who had activity in the last 28 days and their bit patterns.
+WITH submission_date_activity AS (
+  SELECT
+    client_id,
+    days_seen_bits,
+    days_visited_1_uri_bits,
+    days_interacted_bits,
+    submission_date AS activity_date
+  FROM
+    telemetry.clients_last_seen_v1 -- this might cause an issue because definition of first_seen_date in this table is different from clients_first_seen_v2
+  WHERE
+    submission_date = DATE("2023-10-01")
+  GROUP BY
+    client_id,
+    submission_date,
+    days_seen_bits,
+    days_visited_1_uri_bits,
+    days_interacted_bits
+),
+-- Get all the cohorts that are still in range of the current day of activity (180 days)
+cohorts_in_range AS (
+  SELECT
+    client_id,
+    sample_id,
+    first_seen_date AS cohort_date,
```

Comment:
Should we be renaming this? I think it's useful to have consistency between tables. My thinking is this should be kept as `first_seen_date`.

Reply:
Will users understand that first_seen_date is the cohort_date if we don't rename it?